aperturedb/Images.py

```diff
@@ -253,11 +254,12 @@ def __retrieve_batch(self, index):
         end = min(start + self.batch_size, len(self.images_ids))
 
         query = []
+        FindCommand = f"Find{self.query_object}"
```

Comment:
This variable name seems odd case, will not be needed if Querybuilder is used.

Reply:
changed to QueryBuilder